minio/mcs
# Policy screens

# What does this do?
Enable policy list, remove and create

# How to test?
- Pull branch
- run yarn install inside portal-ui folder
- run mcs in port 5000 and review
- go to policies section
- create new policy using the text edit
- verify the policy appears on the list
